src/engine/source/kvdb2/include/kvdb2/iKVDBHandler.hpp

```diff
@@ -2,9 +2,12 @@
 #define _IKVDBHANDLER_H
 
 #include <error.hpp>
+#include <memory>
 #include <string>
 #include <variant>
 
+#include <unordered_map>
```

Comment:
Here we could remove the newline in the middle and sort alphabetically the includes

Reply:
The same goes for all the occurrences